openhtf/io/user_input.py

```diff
@@ -84,7 +85,7 @@ def DisplayPrompt(self, message, text_input=False):
       console_prompt = ConsolePrompt(
           message, functools.partial(self.Respond, self.prompt.id))
       console_prompt.start()
-      self._cond.wait(int(FLAGS.prompt_timeout_s))
+      self._cond.wait(timeout)
```

Comment:
Hmm. Being able to pass ```None``` explicitly is a good argument in favor of doing it @grybmadsci's way.